internal/utils/ocpbuild/helper.go

```diff
@@ -33,7 +33,7 @@ func NewOCPBuildsHelper(client client.Client, buildType string) OCPBuildsHelper
 	}
 }
 
-func (o *ocpBuildsHelper) GetModuleOCPBuildByKernel(ctx context.Context, mld *api.ModuleLoaderData) (*buildv1.Build, error) {
+func (o *ocpBuildsHelper) GetModuleOCPBuildByKernel(ctx context.Context, mld *api.ModuleLoaderData, owner metav1.Object) (*buildv1.Build, error) {
 	buildList := buildv1.BuildList{}
 
 	opts := []client.ListOption{
```

Comment:
Could we use a `fieldSelector` instead of filtering manually?

Reply:
i don't think you can quire using Owner field, field selector is actually limited to which fields you can use